src/mock-doc/test/element.spec.ts

```diff
@@ -479,4 +479,12 @@ describe('element', () => {
       expect(elm.pathname).toBe('/path/to/page');
     });
   });
+
+  describe('ul', () => {
+    it('textContent', () => {
+      const elm: MockUListElement = doc.createElement('ul');
```

Comment:
There aren't any unique properties here to test - so I just went with `textContent` to test the creation of _something_